config-local.yaml

```diff
@@ -0,0 +1,28 @@
+# See https://docs.mitmproxy.org/stable/concepts-options/ for documentation
+
+listen_port: 8080
+web_port: 8081
+
+# Make sure that the right configuration files and certs are loaded
+client_certs: ./client-certificates/WiiU-common.pem
+confdir: ./.mitmproxy
+
+# Enable the Pretendo redirection script
+scripts: ["../pretendo_addon.py"]
+pretendo_redirect: true
+
+# Set up redirection to a local server
+pretendo_host: 127.0.0.1
+pretendo_host_port: 80
+pretendo_http: true
+
+# Allow self-signed certificates
+ssl_insecure: true
+
+# Allow the console to use older TLS versions
+tls_version_client_min: UNBOUNDED
+tls_version_server_min: UNBOUNDED
+
+# A few convenience features
+anticache: true
+showhost: true
```

Comment:
Not sure if these options are needed for Pretendo redirection? I would leave it to user's choice